plugins/metrics/metrics.go

```diff
@@ -187,3 +187,25 @@ func Namespace(ns string) nirvana.Configurer {
 		return nil
 	}
 }
+
+// Plugin contains basic configurations of metrics.
+type Plugin struct {
+	// Namespace is metrics namespace.
+	Namespace string `desc:"Metrics namespace"`
```

Comment:
how about default value?

Reply:
It uses `Namespace` configurer. Defaults to "nirvana".
```go
// Namespace returns a configure to set metrics namespace.
func Namespace(ns string) nirvana.Configurer {
	if ns == "" {
		ns = "nirvana"
	}
	return func(c *nirvana.Config) error {
		wapper(c, func(c *config) {
			c.namespace = ns
		})
		return nil
	}
}
```

Reply:
So you depend on plugin author to know and check input string and provide a default, which is not a user-friendly API.  Since you already provide `desc` as part of option tag, having a `default` tag sounds more natural.